go.mod

```diff
@@ -113,4 +113,4 @@ require (
 	k8s.io/utils v0.0.0-20230406110748-d93618cff8a2 // indirect
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.3.0 // indirect
-)
+)
```

Comment:
Why remove EOL?

Reply:
my editor autoformatted this. I'll remove this